src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -162,7 +162,11 @@ public boolean isDockerFileMode() {
     }
 
     public File getContextDir() {
-        return contextDir != null ? new File(contextDir) : getDockerFile().getParentFile();
+        return contextDir != null
+                ? new File(contextDir)
+                : (getDockerFile().getParentFile() != null
+                     ? getDockerFile().getParentFile()
+                     : new File(Paths.get("").toAbsolutePath().toString()));
```

Comment:
I mean, it should be sufficient to just return ".". It gets converted to an absolute path later.

Reply:
but method is returning a `File`, you mean returning a `new File(".")`?

Reply:
Yeah, that was my idea. But not 100% sure.